File: bddtests/src/main/kotlin/mocking/emis/models/PrescriptionRequest.kt
package mocking.emis.models

data class PrescriptionRequest(var dateRequested: String,
                               var requestedMedicationCourses: MutableList<RequestedMedicationCourse>,
                               var status: String,
                               var requestedByDisplayName: String,
                               var requestedByForenames: String,
                               var requestedBySurname: String)

File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Emis/Models/Prescriptions/PrescriptionRequest.cs
using System;
using System.Collections.Generic;

namespace NHSOnline.Backend.GpSystems.Suppliers.Emis.Models.Prescriptions
{
    public class PrescriptionRequest
    {
        
        public string RequestedByDisplayName { get; set; }
        
        public string RequestedByForenames { get; set; }
        
        public string RequestedBySurname { get; set; }
        
        public DateTimeOffset DateRequested { get; set; }

        public IEnumerable<RequestedMedicationCourse> RequestedMedicationCourses { get; set; }
    }
}
